### PR TITLE
Summit 2016 schedule

### PR DESCRIPTION
**Pull request by tjsavage**
This PR adds the schedule to the Polymer Summit site.

@graynorton - would you mind taking a look at the descriptions and see if anything needs to be added? Particularly the keynote description and tools keynote description.

Thanks!

### CODE REVIEW

File: app/summit-2016/data/schedule.json [hunk 1]
```diff
@@ -1,162 +1,142 @@
...
+        "name":"Opening Keynote",
+        "speaker":"Taylor Savage",
+        "desc":"Welcome to the second Polymer Summit! We'll kick off with a whirlwind tour of what's new in the world of Web Components and the Polymer Project."
+      },
       {
-        "name":"Registration Opens",
-        "time":"Tuesday @ 7:30am"
+        "name": "The Way of the Web - ING",
+        "speaker": "Ben Issa",
+        "desc": "An end-to-end view on how ING builds beautiful experiences for its customers using Web Components."
```
Comment (arthurevans) [inline app/summit-2016/data/schedule.json]: Need to capitalize or not capitalize "web components" consistently in all the descriptions. We have a couple with caps at the top, some with no caps later on.

In the docs, we've tried to go with no caps (unless we're talking about a specification by name). But I don't think consistency with the rest of the docs is as important as consistency across the schedule.

File: app/summit-2016/data/schedule.json [hunk 2]
```diff
@@ -1,162 +1,142 @@
...
       {
-        "name":"Opening Keynote",
-        "speaker":"Matthew McNulty & Taylor Savage",
-        "time":"Tuesday @ 9:00am",
-        "video": "https://www.youtube.com/watch?v=jVn8tlnwAEs&list=PLNYkxOF6rcICdISJclfQhj2S8QZGjXV8J&index=1",
-        "desc":"A warm welcome to the first ever Polymer Summit. We'll talk briefly about how we're thinking about Polymer's past, present, and future, and get the show on the road.",
-        "link": "/summit/sessions/0"
+        "name": "Polymer 2.0: Under the Hood",
+        "speaker": "Rob Dodson",
+        "desc": "It's not always clear to developers the boundaries between native Web Components and Polymer. What does the library add? What's already built into the platform? This talk aims to demystify Polymer by peeling back the cover to explain how the internals work and show how they complement existing APIs. We'll cover the basics of building vanilla Web Components using the new v1 specs, and demonstrate new features in Polymer 2.0 that make the whole process more efficient."
```
Comment (arthurevans) [inline app/summit-2016/data/schedule.json]: Web components--capitalized here.

File: app/summit-2016/data/schedule.json [hunk 3]
```diff
@@ -1,162 +1,142 @@
...
+        "name": "An app in 3 days or: how I learned to stop worrying and love the Polymer toolbox",
+        "speaker": "Monica Dinculescu",
+        "desc": "Think of your favorite app. It's probably meowni.ca/emojillate (who can blame you), but if it isn't (and we need to talk) it probably has one, if not all, of these things: a database; a potential for disaster, with many users looking at and clicking on the same data; offline support, a responsive layout while preferably being faster than the average bear. Recreating this experience from scratch might sound like a terrifying experience, but I'm going to show you how the polymer toolbox and a few other already-built elements can help you go from zero to hero and build a PWA in no time."
```
Comment (arthurevans) [inline app/summit-2016/data/schedule.json]: Second sentence is a run-on and quite hard to read: 

"It's probably meowni.ca/emojillate (who can blame you), but if it isn't (and we need to talk) it probably has one, if not all, of these things: a database; a potential for disaster, with many users looking at and clicking on the same data; offline support, a responsive layout while preferably being faster than the average bear."

Recommend fixing the list so it has an end, and making it two sentences:

"... offline support; and a responsive layout. While preferably being faster than the average bear."

Or add performance to the list. "offline support; better than average performance; and a responsive layout." (Admittedly, that cuts one joke.)

Second sentence:
"Recreating this experience from scratch might sound like a terrifying experience, but I'm going to show you how the polymer toolbox and a few other already-built elements can help you go from zero to hero and build a PWA in no time."

"experience" is repeated—just dropping the first experience sounds fine to me ("Recreating this from scratch"), but "Recreating this app from scratch" or "Creating an app like this from scratch" would also work. "polymer toolbox" should be "Polymer App Toolbox."

File: app/summit-2016/data/schedule.json [hunk 4]
```diff
@@ -1,162 +1,142 @@
...
-    "day":"Tuesday",
-    "sessions":[
+        "name":"Opening Keynote",
+        "speaker":"Taylor Savage",
+        "desc":"Welcome to the second Polymer Summit! We'll kick off with a whirlwind tour of what's new in the world of Web Components and the Polymer Project."
+      },
       {
-        "name":"Registration Opens",
-        "time":"Tuesday @ 7:30am"
+        "name": "The Way of the Web - ING",
```
Comment (arthurevans) [inline app/summit-2016/data/schedule.json]: Throughout: In the docs, we always use em dashes with no space: "The Way of the Web-ING" In the case of ING, that looks a little weird... Could use "The Way of the Web: ING"?

**Comment by tjsavage** [comment]
Thanks @arthurevans! Pushed fixes to issues you mention

**Pull request merged**